### ISSUE TITLE
update logo

### ISSUE THREAD

**Issue by MelSumner** [ember-a11y/a11y-demo-app#16]
The ember-a11y project has obtained an Ember mascots since creating this website. The logo should be replaced with the A11y Zoey and Tomster.
